README.md

```diff
@@ -1,2 +1,88 @@
-# hexabot-plugin-chatgpt
-The OpenAI ChatGPT Plugin for Hexabot Chatbot / Agent Builder to enable the LLM RAG Capability
+# ChatGPT RAG Plugin for Hexabot Chatbot Builder
+
+The ChatGPT RAG Plugin integrates OpenAI’s GPT-4 model with Hexabot chatbot workflows to enable Retrieval-Augmented Generation (RAG) capabilities. This plugin allows users to enrich their chatbot responses using relevant content retrieved from a content management system, making interactions more contextual and accurate.
+
+[Hexabot](https://hexabot.ai/) is an open-source chatbot / agent solution that allows users to create and manage AI-powered, multi-channel, and multilingual chatbots with ease. If you would like to learn more, please visit the [official github repo](https://github.com/Hexastack/Hexabot/).
+
+## Prerequisites
+
+Before setting up the ChatGPT RAG Plugin, you will need to generate an API token from OpenAI.
+
+1. Go to the [OpenAI API page](https://openai.com/api).
+2. Obtain your API key.
+3. Once you have your API key, you can configure the plugin in Hexabot.
+
+## Installation
+
+First, navigate to your Hexabot project directory and make sure the dependencies are installed:
+
+```sh
+cd ~/projects/Hexabot
+npm install
+```
+
+To install the ChatGPT RAG Plugin, run the following command:
+
+```sh
+npx hexabot install plugin Hexastack/hexabot-plugin-chatgpt-rag
```

Comment:
This is the name of the repo : 
```suggestion
npx hexabot install plugin Hexastack/hexabot-plugin-chatgpt
```

Reply:
Okay updated :heavy_check_mark: 
Thank you :pray: